e2e/resources/workbench-config.ts

```diff
@@ -31,7 +31,9 @@ const local: IEnvConfig = {
   ADMIN_TEST_USER: process.env.ADMIN_TEST_USER || 'puppeteer-admin-1@fake-research-aou.org',
   DEFAULT_CDR_VERSION_NAME: 'Synthetic Dataset v3',
   ALTERNATIVE_CDR_VERSION_NAME: 'Synthetic Dataset v3 with WGS',
```

Comment:
What are the requirements around ALTERNATIVE_CDR_VERSION_NAME, just that it's an older version? If so, can we rename it to something like OLD_CDR_VERSION_NAME.

I will need to send another config update - my most recent one archived all but two versions, and I think we now need 3 to support puppeteer